orgakram/site.yml

```diff
@@ -0,0 +1,8 @@
+---
+- hosts: poolrechner
+  roles:
+    - pool-setup
```

Comment:
ich glaube es macht sinn diese beiden schritte in separate playbooks zu separierei 🤔

Reply:
ja, ist wahrscheinlich sauberer